pkg/virt-launcher/virtwrap/device/sriov/hostdev.go

```diff
@@ -62,6 +64,7 @@ func createHostDevice(iface v1.Interface, hostPCIAddress string) (*api.HostDevic
 		return nil, fmt.Errorf("failed to create SRIOV device for %s, host PCI: %v", iface.Name, err)
 	}
 	hostDev := &api.HostDevice{
+		Alias:   &api.Alias{Name: AliasPrefix + iface.Name},
```

Comment:
Well hostdev_test.go can access private function since it's under the same package, If you mean that we should not use private functions from the package at the test suite, then it make sense, still having a small function with this logic is not a bad thing per se, but up to you man.

Reply:
It is under the `sriov_test` package. I consider it a bad practice to have the tests and production code in the same package for this exact reason. Testing private functions makes the tests fragile and prone to fail if inner implementation changes where the API is kept stable.